website/addons/box/settings/__init__.py

```diff
@@ -0,0 +1,7 @@
+import logging
+from .defaults import *  # noqa
+
+try:
+    from .local import *  # noqa
+except ImportError as error:
+    logging.warn('No local.py settings file found')
```

Comment:
Use a named logger instead of the root logger.

```python
import logging
from .defaults import *  # noqa

logger = logging.getLogger(__name__)

try:
    from .local import *  # noqa
except ImportError as error:
    logger.warn('No local.py settings file found')
```

Will make this change during merge.